edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/Microsoft.Azure.Devices.Edge.Agent.IoTHub.csproj

```diff
@@ -9,8 +9,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Devices" Version="1.28.1-NestedEdge" />
-    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.33.1-NestedEdge" />
+    <PackageReference Include="Microsoft.Azure.Devices" Version="1.22.5-NestedEdge" />
```

Comment:
I was expecting the version to be 1.28.2-NestedEdge and 1.33.2-NestedEdge. We need to confirm with Richard.